person/src/homeworks/person/core/model/BasePerson.java

```diff
@@ -0,0 +1,55 @@
+package homeworks.person.core.model;
+
+import homeworks.person.core.service.PersonAction;
```

Comment:
@tatevikb jan model should be pure object , and actions are services , so you need service package where you will store interfaces for action and in service package impl package to do exact implementations